sdk/servicebus/service-bus/package.json

```diff
@@ -20,9 +20,9 @@
   "module": "dist-esm/src/index.js",
   "browser": {
     "./dist/index.js": "./browser/service-bus.js",
-    "./dist-esm/test/utils/aadUtils.js": "./dist-esm/test/utils/aadUtils.browser.js",
     "./dist-esm/src/util/crypto.js": "./dist-esm/src/util/crypto.browser.js",
     "./dist-esm/src/util/parseUrl.js": "./dist-esm/src/util/parseUrl.browser.js",
+    "./dist-esm/test/utils/aadUtils.js": "./dist-esm/test/utils/aadUtils.browser.js",
```

Comment:
Any particular reason this line was moved?

Reply:
Not really, moved it back .. now that I think about it. The order would have been correct alphabetically, but nvm